src/systemctl/systemctl-logind.c

```diff
@@ -152,9 +152,9 @@ int logind_check_inhibitors(enum action a) {
                 return 0;
 
         r = acquire_bus_full(BUS_FULL, /* graceful = */ true, &bus);
-        if (r == -ECONNREFUSED && geteuid() == 0)
-                return 0; /* When D-Bus is not running, allow root to force a shutdown. E.g. when running at
-                           * the emergency console. */
+        if (ERRNO_IS_NEG_DISCONNECT(r) && geteuid() == 0)
+                return 0; /* When D-Bus is not running (ECONNREFUSED) or D-Bus socket is not created (ENOENT),
+                           * allow root to force a shutdown. E.g. when running at the emergency console. */
```

Comment:
hmm, ENOENT is not included in ERRNO_IS_NEG_DISCONNECT()... hence not grokking this?

Reply:
Interestingly, _it_ does include ENOENT.

Reply:
eheh it's `ENONET` that is included, not `ENOENT` - and yes I misread it too when I reviewed it

Reply:
Oh......